oaiharvest/harvest.py

```diff
@@ -500,16 +520,41 @@ def parse_time(argument):
 # Set up logger
 logging.basicConfig(
     level=logging.DEBUG,
-    format='%(asctime)s %(name)-16s %(levelname)-8s %(message)s',
-    datefmt='[%Y-%m-%d %H:%M:%S]',
-    filename=os.path.join(appdir, 'harvest.log'))
-
-ch = logging.StreamHandler()
+    format='%(levelname)-8s %(message)s',
+    # format='%(asctime)s %(name)-16s %(levelname)-8s %(message)s',
+    # datefmt='[%Y-%m-%d %H:%M:%S]',
+    # filename=os.path.join(appdir, 'harvest.log')
+    )
+
+#ch = logging.StreamHandler()
+ch = logging.FileHandler( os.path.join( appdir, 'harvest.log'))
 ch.setLevel(logging.DEBUG)
-formatter = logging.Formatter('%(levelname)-8s %(message)s')
+formatter = logging.Formatter(
+    '%(asctime)s %(name)-16s %(levelname)-8s %(message)s',
+    '[%Y-%m-%d %H:%M:%S]')
+#formatter = logging.Formatter('%(levelname)-8s %(message)s')
```

Comment:
Please remove old code instead of commenting it out

Reply:
OK: commented code removed. ( I had left them in originally, as I was unsure of the intent of the original code, and wanted to make clear what exactly what behavior I was changing. )
I also added the --recover option to the doc-string at the top, which I missed earlier.